host/xtest/Makefile

```diff
@@ -10,6 +10,10 @@ endif
 # be used instead.
 OPTEE_CLIENT_EXPORT ?= ../../../optee_client/out/export
 
+# If CFG_XTEST_LITE=n : compile full xtest suite (default)
+# If CFG_XTEST_LITE=y : compile sanity xtest suite
+CFG_XTEST_LITE ?= n
```

Comment:
+1 with @jenswi-linaro. IMO compile-time options are inconvenient, so they should not be used unless there is a reason (like reducing file size, or incompatible options...).
It is just a matter of replacing `#ifndef ... #endif` with `if (...) { ... }`, no?